typescript/sdk/src/hook/HyperlaneHookDeployer.ts

```diff
@@ -87,7 +87,11 @@ export class HyperlaneHookDeployer extends HyperlaneDeployer<
       hook = await this.deployRouting(chain, config, coreAddresses);
     } else if (config.type === HookType.PAUSABLE) {
       hook = await this.deployContract(chain, config.type, []);
-      await this.transferOwnershipOfContracts(chain, config.owner, { hook });
+      await this.transferOwnershipOfContracts<HookType.PAUSABLE>(
+        chain,
+        config,
```

Comment:
don't like this, passing unnecessary config to the transferOwnershipOfContracts function which only needs owner